web/gui/main.js

```diff
@@ -1786,21 +1786,26 @@ function renderPage(menus, data) {
         html += mhead + shtml + '</div></div><hr role="separator"/>';
     }
 
+    const isMemoryModeDbEngine = data.memory_mode === "dbengine";
+
     sidebar += '<li class="" style="padding-top:15px;"><a href="https://github.com/netdata/netdata/blob/master/docs/Add-more-charts-to-netdata.md#add-more-charts-to-netdata" target="_blank"><i class="fas fa-plus"></i> add more charts</a></li>';
     sidebar += '<li class=""><a href="https://github.com/netdata/netdata/tree/master/health#Health-monitoring" target="_blank"><i class="fas fa-plus"></i> add more alarms</a></li>';
     sidebar += '<li class="" style="margin:20px;color:#666;"><small>Every ' +
       ((data.update_every === 1) ? 'second' : data.update_every.toString() + ' seconds') + ', ' +
-      'Netdata collects <b>' + data.dimensions_count.toLocaleString() + '</b> metrics, presents them in <b>' +
-      data.charts_count.toLocaleString() + '</b> charts and monitors them with <b>' +
-      data.alarms_count.toLocaleString() + '</b> alarms. Netdata is using ' +
-      Math.round(data.rrd_memory_bytes / 1024 / 1024).toLocaleString() + ' MB of memory on <b>' +
-      data.hostname.toString() + '</b> for ' +
-      NETDATA.seconds4human(data.update_every * data.history, {
-        minute: 'minute', minutes: 'minutes', second: 'second', seconds: 'seconds', space: '&nbsp;',
-      }) +
-      ' of real-time history.<br />&nbsp;<br />' + 'Get more history by ' +
-      '<a href="https://docs.netdata.cloud/docs/configuration-guide/#increase-the-metrics-retention-period" target=_blank>configuring Netdata\'s <b>history</b></a> or using the <a href="https://docs.netdata.cloud/database/engine/" target=_blank>DB engine.</a>' +
-      '<br/>&nbsp;<br/><b>netdata</b><br/>' + data.version.toString() + '</small></li>';
+      'Netdata collects <b>' + data.dimensions_count.toLocaleString() + '</b> metrics on ' +
```

Comment:
Use of the `b` tag is discouraged in the HTML5 spec. Please replace with `strong` or `em` if you want to emphasize.

Reply:
Considering @cosmix last comment, would not be better to use CSS to do this?

Reply:
@thiagoftsm you're absolutely right, however in our case there's no separate styles for this, just bootstrap.css adds
```
b,
strong {
  font-weight: bold;
}
```
Done, but i've only pushed to the modified fragment. There are 20 more cases in main.js, and even if i'm 99% sure nothing bad would happen, i would prefer to change it on refractor phase (also for rebase reasons)

Reply:
tested in chrome, safari, ff

Reply:
Thank you by the explanation!